schema.md

```diff
@@ -0,0 +1,577 @@
+# WES - WRC RO-Crate
+
+This document aims to detail a conversion between the [GA4GH WES profile](https://ga4gh.github.io/workflow-execution-service-schemas/docs/) to the [Workflow Run Crate](https://www.researchobject.org/workflow-run-crate/profiles/workflow_run_crate/) (WRC) extension of [Ro-Crates](https://www.researchobject.org/ro-crate/)
+
+## WORKFLOW RUN CRATE RO-CRATE FIELDS
+
+For a Json object to conform with the Ro-Crate specification there are certain minimal requirements. Further still the WRC estension has it's own requirements. Considering that, the following definitions aim to differentiate the diffrent types of fields and how they should be considered in each implementation.
+
+### Terminology
+
+1. constant: These fields are required and for our purposes the value of this field should always be the default value. In other words their values are constant.
+2. required: These fields are required but their values are not constant.
+3. recommended: These fields are not required but they are recommended to give a comprehensive definition for your RO-crate object.
+4. optional: These fields are optional.
+
+### `base object`
+
+This is the base Ro-Crate object.
+
+- **@context**(constant):
+  - **WES field**: N/A
```

Comment:
Perhaps you could distinguish between "N/A" meaning "not applicable" (i.e., we don't need WES here, it doesn't apply - so no need to think about the mapping; which is the case here) and "not available" (i.e., we would need this value from somewhere, according to whether it's required, recommended, or optional - but WES doesn't supply it).

These "WES field: not available" properties would be the most tricky ones!

Reply:
I guess a clarification would be beneficial, but what N/A currently means throughout this document is "not applicable". Because for all the required Ro-Crate properties that would be "not avaible" i found that it has been very trivial to get a suitable value, as they are usually descriptive, like the name of the description of the property. For that reason I didn't see a reason to distinguish them.

Reply:
Perhaps after we better distinguish which of these scenarios best fits the "not applicable" description then I can adjust the documentation.

Reply:
I think that for every value, we should try to come up with a possible source (WES, TES or otherwise), or multiple (see below) - and that should be our focus. Of course, within those, required properties MUST, recommended SHOULD and optional MAY have a source (or more).

So I guess what I'm trying to say is that maybe we don't need to think too much about what it means with respect to WES, but rather what the best source will be. Sometimes it's obvious (that could be the constants you are talking about, or a really good field from WES, or something that can be generated on the fly), sometimes it's not so obvious. In some cases, it might be a compound, or something that may or may not be present, depending on the context (e.g., what exactly the workflow run was like). For these, we should list all the possible sources that could help, and possibly define a priority of sources (e.g., if we have _this_, then that'd be the best, but if we don't, maybe this _other_ thing is the next best option).

In the end we really need this summary table that outlines exactly that - an actionable mapping of sources for each WRROC property, i.e., precise enough that it can be implemented.

Reply:
Basically [what we already have](https://docs.google.com/spreadsheets/d/1JP9p2npUiQnt3a2hNeJqjfom8rcB7BDM4SV9HYcIu58/edit?usp=sharing), but with more details (it's better for TES).

Also: Not sure if you are on the official RO-Crate Slack. There were some issues discussed that other people had with mapping WES to WRROC a few months ago - and I don't think we ever picked them up or solved them. I'll share a link when I find the time next week.